vms/authentication/views.py

```diff
@@ -1,10 +1,11 @@
 from django.shortcuts import redirect
+from django.core.urlresolvers import reverse
 from django.http.response import HttpResponse
 from vms import settings
 
 
 def index(request):
-    return HttpResponse("Hello world")
+    return redirect(reverse('authentication:login_process'))
 
```

Comment:
Shouldn't this be HttpResponseRedirect(reverse('authentication:login_process'))?